src/ClientHandle.cpp

```diff
@@ -1188,7 +1188,14 @@ void cClientHandle::HandleLeftClick(int a_BlockX, int a_BlockY, int a_BlockZ, eB
 
 		case DIG_STATUS_SWAP_ITEM_IN_HAND:
 		{
-			// TODO: Not yet implemented
+
+			cItem EquippedItem = m_Player->GetEquippedItem();
+			cItem OffhandItem = m_Player->GetOffHandEquipedItem();
+
+			cInventory & Intentory = m_Player->GetInventory();
+			Intentory.SetShieldSlot(EquippedItem);
```

Comment:
This still contains 'Shield'. Perhaps `SetOffHandSlot`?

Reply:
@NiLSPACE  That function is already exported to the lua API and we don't want to break compatibility.
You can change it if you want (without breaking compatibility) but I think that it is out of scope for this PR.

Reply:
Oh, I see. Perhaps we can rename `SetShieldSlot` and create a manualbinding to the Lua API for backwards compatibility, but that's something for later on :)